client/python/gradio_client/serializing.py

```diff
@@ -111,15 +114,13 @@ def example_inputs(self) -> Dict[str, Any]:
 class BooleanSerializable(Serializable):
     """Expects a boolean as input/output but performs no serialization."""
 
-    def api_info(self) -> Dict[str, List[str]]:
+    def api_info(self) -> Dict[str, Dict | bool]:
         return {
-            "raw_input": ["bool", "boolean value"],
-            "raw_output": ["bool", "boolean value"],
-            "serialized_input": ["bool", "boolean value"],
-            "serialized_output": ["bool", "boolean value"],
+            "info": serializer_types["BooleanSerializable"],
+            "serialized_info": False,
```

Comment:
I think we can get rid of `serialized_info`

Reply:
Cool I was wondering about that as well. Why some components have it and why others don't